mbed_host_tests/host_tests_plugins/module_reset_mps2.py

```diff
@@ -27,15 +29,15 @@ class HostTestPluginResetMethod_MPS2(HostTestPluginBase):
 
     @details Supports:
              reboot.txt   - startup from standby state, reboots when in run mode.
-             shutdown.txt - shutdown from run mode.
-             reset.txt    - reset FPGA during run mode.
+             shutdown.txt - not implemented.
+             reset.txt    - not implemented.
     """
 
     # Plugin interface
     name = 'HostTestPluginResetMethod_MPS2'
     type = 'ResetMethod'
     capabilities = ['reboot.txt', 'shutdown.txt', 'reset.txt']
```

Comment:
From our previous conversation, would you mind removing `shutdown.txt` and `reset.txt`? Currently there's nothing htrun that would make use of these capabilities.